ReferenceDeployment/Top/ReferenceDeploymentTopology.cpp

```diff
@@ -171,20 +173,34 @@ void setupTopology(const TopologyState& state) {
 Os::Mutex cycleLock;
 volatile bool cycleFlag = true;
 
+void myIsr(U32 ticks) {
+    ReferenceDeployment::blockDrv.callIsr(ticks);
+}
+
```

Comment:
Add an `extern C` function here that is callable from the target shell. It can call `blockDrv.stopTimer()` and halt the timer. The C function can set a flag to stop the wait loop on line 194.